src/cluster.ts

```diff
@@ -142,11 +201,15 @@ Please use the format 'my-cluster' or '${instance.name}/clusters/my-cluster'.`);
    * @example <caption>include:samples/document-snippets/cluster.js</caption>
    * region_tag:bigtable_delete_cluster
    */
-  delete(gaxOptions, callback) {
-    if (is.fn(gaxOptions)) {
-      callback = gaxOptions;
-      gaxOptions = {};
-    }
+  delete(
+      gaxOptionsOrCallback?: CallOptions|DeleteClusterCallback,
+      cb?: DeleteClusterCallback): void|Promise<ApiResponse> {
+    const callback =
+        typeof gaxOptionsOrCallback === 'function' ? gaxOptionsOrCallback : cb!;
+    const gaxOptions =
+        typeof gaxOptionsOrCallback === 'object' && gaxOptionsOrCallback ?
```

Comment:
I add `&& gaxOptionsOrCallback` to filter for `null` - I'm thinking it's nice to have but I'd understand it not being necessary